nw-firestore-schema.ts

```diff
@@ -101,7 +101,17 @@ interface FAQ {
   lastModifiedBy: string;
 }
 
+interface CMS {
+  CMSOff: boolean;
```

Comment:
nit: `status`, `cmsStatus`, `maintenanceMode`  ? sorry im being rly picky

Reply:
just using the date now